Fix publish process in docs (#466)
vtex-apps/io-documentation: docs/en/Recipes/store/making-your-new-app-version-publicly-available.md

# Making your new app version publicly available

If you’re comfortable with the configurations you’ve performed and want your version of the app to be made available to any user, [linking](https://vtex.io/docs/recipes/store/linking-an-app) the app in which you’re working will not suffice: you’ll also need to **release** it, **publish** a release candidate, install it in a **production workspace**, **validate** your release candidate and promote it to **master**.

## Releasing

If you’re **sure about all the changes** that you’ve done to your development workspace, it’s time to **release** a new version of your app.

> We’ll use the `vtex release {major/minor/patch}` command to release the version in the app’s `manifest.json` according to the SemVer (semantic versioning) best practices, updating the app's CHANGELOG.md, assigning commit tags and sending the performed changes to the app's repository.

Reviewer: Eu chequei novamente o comando para release e percebi que eu passei uma informação levemente errada -
```vtex release {major/minor/patch} stable``` vai incrementar a versão no `manifest.json` como no exemplo
![Screenshot from 2019-12-16 15-21-06](https://user-images.githubusercontent.com/26463288/70932116-bcaccf80-2017-11ea-8149-a0ae0163f813.png)

Para lançar uma versão beta está correto:
![Screenshot from 2019-12-16 15-22-25](https://user-images.githubusercontent.com/26463288/70932187-e36b0600-2017-11ea-8859-974ea858fe00.png)

Sobre essa linha, eu acho que talvez poderia trocar:
```suggestion
We’ll use the `vtex release {major/minor/patch}` command to increment the version in the app’s `manifest.json` according to the SemVer (semantic versioning) best practices, update the app's `CHANGELOG.md`, create a release commit and a release tag and finally send the performed changes to the app's repository.
```
O que acha?

> If you desire to release a beta version, you should run `vtex release {major/minor/patch} beta` in your terminal. This command will perform the same activities as the previous one, the difference being that you’ll be launching the app’s beta version.

Reviewer: O que acha de
```suggestion
If you desire to release a beta version, you should run `vtex release {major/minor/patch} beta` in your terminal. This command will perform the same tasks as the previous one, the difference being that you’ll be launching a beta version for your app.
```

<div class="alert alert-warning">
You must be logged into the account where you want the new app version to be released. Make sure the app’s <code>vendor</code> is <strong>equal</strong> to its <code>account</code>.
</div>

## Publishing

> Once a app version was released, the app **must be installed** in order to test its new settings. However, it is not possible to install an app that only exists in your local environment. You must **publish** the app so that it can be installed by yourself or by others as well.

Reviewer: ```suggestion
Once an app version is released, the app **must be installed** in order to test its new settings. However, it is not possible to install an app that only exists in your local environment. You must **publish** the app so that it can be installed by yourself or by others as well.
```
Estou em dúvida se deveria ser ```Once an app's version is released``` ou ```Once an app version is released```

Reviewer: Que acha de `behavior`? 
```suggestion
Once a app version was released, the app **must be installed** in order to test its new behavior. However, it is not possible to install an app that only exists in your local environment. You must **publish** the app so that it can be installed by yourself or by others as well.
```

Reviewer: Acho que poderia sim. Eu estava pensando no caso em que um desenvolvedor faz uma app de backend e ele muda algo diretamente no código, o que muda o comportamento da app, acho que não necessariamente configuração, mas lembrei agora também dos casos de mudanças nos blocks/interfaces, que daí seriam mudanças de settings mesmo.

> Use the `vtex publish` command to turn the account version in which you were working into the app’s **release candidate version** for installment. Notice: a release candidate can **only** be installed on an account for testings if the user orders Toolbelt to install the exact version.

Reviewer: Não entendi muito bem o início da frase:
```
Use the `vtex publish` command to turn the account version in which you were working into the app’s **release candidate version** for installment.
```

Reviewer: Boa, acho que ficou melhor também

<div class=“alert alert-warning”>

> You always must be logged into the account where you want the new app version to be published. Make sure the app’s <code>vendor</code> is <b>equal</b> to its <code>account</code>.

Reviewer: Sobre essa parte:
```
Make sure the app’s <code>vendor</code> is <b>equal</b> to its <code>account</code>.
```
O toolbelt antes de publicar checa se o usuário está na mesma conta que a especificada no vendor da app e se não estiver ele pede para trocar:

![Screenshot from 2019-12-16 15-47-37](https://user-images.githubusercontent.com/26463288/70933910-8d985d00-201b-11ea-8c6e-28ff62dbca08.png)

Se o usuário responder que não deseja trocar ele não consegue publicar, ele só consegue publicar se trocar

Reviewer: Agora que percebi que essa mensagem de erro não está legal, seria melhor se falasse `from the vender indicated on your manifest.json`, algo assim :/

Author: Ah, boa! Achei que não rolava esse impedimento pelo Toolbelt. Se rola, vamo tirar essa mensagem porque fica poluição visual só. 

"You are trying to publish this app in an account that differs from the vendor indicated in the app's manifest.json. Log in to XXXXXX to publish your app in the correct account?" ->>> Acho que a gente pode mudar o do you want pra uma mensagem menos amigável, porque o do you want dá a entender pro usuário que ele pode publicar se responder que yes. Mas esse não é o cenário, né?

Reviewer: > "You are trying to publish this app in an account that differs from the vendor indicated in the app's manifest.json. Log in to XXXXXX to publish your app in the correct account?" ->>> Acho que a gente pode mudar o do you want pra uma mensagem menos amigável, porque o do you want dá a entender pro usuário que ele pode publicar se responder que yes. Mas esse não é o cenário, né?

Boa, o usuário só pode publicar se ele logar na conta correta, então não faz mesmo sentido perguntar se ele quer dar publish nessa account, realmente dá impressão que ele pode fazer o publish sem mudar de conta. Vou marcar isso pra mudar

</div>

If your app **doesn’t** have billing options, users with access to the account in which the app was published can install it through the Admin’s Apps section. Otherwise, the app can be installed using Toolbelt, regardless of the specified [billing method](http://help.vtex.com/en/tutorial/app-pricing-models--2ZKBKxLe08Q6seA6sCi6o2).

## Production workspace

> Now that you candidate release version is ready to be installed, you must **create a workspace in production mode** using the following command in order to test the app version settings:

Reviewer: Que tal:
```suggestion
Now that your release version candidate is ready to be installed, you must **create a workspace in production mode** in order to test this app's version behavior. To create it use the following command:
```

Author: O termo release version candidate tá certo mesmo pelo time de localization. E quanto ao settings/behavior, eu falei antes também.

```sh
vtex use {{WorkspaceName}} --production
```

<div class="alert alert-warning">
From this point onwards, any changes to the code are prohibited in the workspace, once it is ready to receive traffic, that is, to be accessed. If you want to change your code, work on it in a developer workspace and then follow the above-mentioned steps again.
</div>

> ### Instaling the candidate release version

Reviewer: ```suggestion
### Installing the candidate release version
```
Não sei o que ficaria melhor: ```release version candidate``` ou ```candidate release version```

Author: Release version candidate é o mais correto mesmo.

> To install the candidate release in the new production workspace, simply run one of the following commands:

Reviewer: ```suggestion
To install the release candidate in the new production workspace, simply run one of the following commands:
```

```sh
`vtex install {appvendor}.{appname}@{appversion}` to install a specific version of the app.
```

<div class="alert alert-info">

>   If you are releasing a <strong>beta</strong> app version and all settings were already tested by you, you should release a non-beta version for your app, publish it and install it in a production workspace, as stated previously. 

Reviewer: ```suggestion
  If you are releasing a <strong>beta</strong> version for your app and the new behavior was already tested by you, you should release a non-beta version, publish it and install it in a production workspace, as stated previously. 
```

Author: ok. Só adicionaria settings também próximo a behavior.

</div>

## Validating

> Once the version is already installed in a production workspace, it is time to **validate your candidate release**. 

Reviewer: ```suggestion
Once the version is already installed in a production workspace, it is time to **validate your release candidate**. 
```

Author: mesma coisa

1. Test the release candidate’s **stability** by means of an [A/B test](https://vtex.io/docs/recipes/store/running-native-ab-testing).

<div class="alert alert-info">

>   If you are releasing a <strong>beta</strong> app version and all settings were already tested by you, you should release a non-beta version for your app, publish it, install it in a production workspace and test its stability once more in order to follow the next step. 

Reviewer: ```suggestion
  If you are releasing a <strong>beta</strong> version for your app and the new behavior was already tested by you, you should release a non-beta version, publish it, install it in a production workspace and test its stability once more in order to follow to the next step. 
```

Author: mesma coisa

</div>

> 2. Use the `vtex validate` command to **publish the release candidate as a stable version**. By using this command, Housekeeper will automatically install the new app stable version on all accounts that use the app.

Reviewer: Eu não sei se os desenvolvedores parceiros/clientes conhecem o serviço 'Housekeeper', talvez isso seja algo interno mesmo. Podemos abstrair essa parte e falar algo como:
```suggestion
2. Use the `vtex validate` command to **publish the release candidate as a stable version**. By using this command, your new app stable version will be automatically installed on all accounts that use your app.
```

Author: Boa! Também acho desnecessário. Só confunde e nao agrega nada indicar que é o Housekeeper.
